doc/managing_pachyderm/general_troubleshooting.md

```diff
@@ -0,0 +1,227 @@
+# General Troubleshooting
+
+Here are some common issues by symptom along with steps to resolve them.  They are organized into the following categories:
+
+- [Deploying a Pachyderm cluster](#starting-up-a-pachyderm-cluster)
+- [Connecting to a Pachyderm cluster](#connecting-to-a-pachyderm-cluster)
+- [Problems running pipelines](#problems-running-pipelines)
+
+---
+
+## Deploying A Pachyderm Cluster
+
+- [Pod stuck in `CrashLoopBackoff`](#pod-stuck-in-crashloopbackoff)
+- [Pod stuck in `CrashLoopBackoff` - with error attaching volume](#pod-stuck-in-crashloopbackoff-with-error-attaching-volume)
+
+### Pod stuck in `CrashLoopBackoff`
+
+#### Symptoms
+
+The pachd pod keeps crashing/restarting:
+
+```
+$ kubectl get all
+NAME                        READY     STATUS             RESTARTS   AGE
+po/etcd-281005231-qlkzw     1/1       Running            0          7m
+po/pachd-1333950811-0sm1p   0/1       CrashLoopBackOff   6          7m
+
+NAME             CLUSTER-IP       EXTERNAL-IP   PORT(S)                       AGE
+svc/etcd         100.70.40.162    <nodes>       2379:30938/TCP                7m
+svc/kubernetes   100.64.0.1       <none>        443/TCP                       9m
+svc/pachd        100.70.227.151   <nodes>       650:30650/TCP,651:30651/TCP   7m
+
+NAME           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deploy/etcd    1         1         1            1           7m
+deploy/pachd   1         1         1            0           7m
+
+NAME                  DESIRED   CURRENT   READY     AGE
+rs/etcd-281005231     1         1         1         7m
+rs/pachd-1333950811   1         1         0         7m
+```
+
+#### Recourse
+
+First describe the pod:
+
+```
+$ kubectl describe po/pachd-1333950811-0sm1p
+```
+
+If you see an error including `Error attaching EBS volume` or similar, see the recourse for that error here under the corresponding section below this one.  If you don't see that error, but do see something like:
+
+```
+  1m    3s    9    {kubelet ip-172-20-48-123.us-west-2.compute.internal}                Warning    FailedSync    Error syncing pod, skipping: failed to "StartContainer" for "pachd" with CrashLoopBackOff: "Back-off 2m40s restarting failed container=pachd pod=pachd-1333950811-0sm1p_default(a92b6665-506a-11e7-8e07-02e3d74c49ac)"
+```
+
+That means Kubernetes tried running `pachd`, but `pachd` generated an internal error. To see the specifics of this internal error, check the logs for the `pachd` pod:
+
+```
+$kubectl logs po/pachd-1333950811-0sm1p
+```
```

Comment:
Note: if you're using a log aggregator service (e.g. the default in GKE) you won't see any logs this way, but will need to look at your logs UI (e.g. in GKE's case the stackdriver console)